config/environments/test.rb

```diff
@@ -73,7 +73,7 @@
   # enable Bullet in testing mode only if requested
   config.after_initialize do
     Bullet.enable = true
-    Bullet.raise = true
+    Bullet.raise = false
```

Comment:
I'm not sure if there' a new added feature for this gem but with Rails 7 it started failing test. Will revisit back after Rail 7 upgrade.